frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -67,65 +89,47 @@ const ExpressionWidget = (props: ExpressionWidgetProps): JSX.Element => {
             }}
             onError={(errorMessage: string) => setError(errorMessage)}
           />
-          <p className="h5 text-medium">
-            {t`Think of this as being kind of like writing a formula in a spreadsheet program: you can use numbers, fields in this table, mathematical symbols like +, and some functions. So you could type something like Subtotal - Cost.`}
-            &nbsp;
-            <ExternalLink
-              className="link"
-              target="_blank"
-              href={MetabaseSettings.docsUrl(
-                "questions/query-builder/expressions",
-              )}
-            >{t`Learn more`}</ExternalLink>
-          </p>
         </div>
+      </ExpressionFieldWrapper>
+      <FieldWrapper>
+        <FieldTitle>{t`Name`}</FieldTitle>
+        <Input
+          type="text"
+          value={name}
+          placeholder={t`Something nice and descriptive`}
+          fullWidth
+          onChange={event => setName(event.target.value)}
+          onKeyPress={e => {
+            if (e.key === "Enter") {
+              handleCommit();
+            }
+          }}
+        />
+      </FieldWrapper>
 
-        <div className="mt3 h5 text-uppercase text-light text-bold">{t`Name`}</div>
-        <div>
-          <input
-            className="my1 input block full"
-            type="text"
-            value={name}
-            placeholder={t`Something nice and descriptive`}
-            onChange={event => setName(event.target.value)}
-            onKeyPress={e => {
-              if (e.key === "Enter") {
-                handleCommit();
-              }
-            }}
-          />
-        </div>
-      </div>
+      <Divider />
+      <Footer>
+        <ActionButtonsWrapper>
+          <Button onClick={() => onClose && onClose()}>{t`Cancel`}</Button>
 
-      <div className="mt2 p2 border-top flex flex-row align-center justify-between">
-        <div className="ml-auto">
-          <button
-            className="Button"
-            onClick={() => onClose && onClose()}
-          >{t`Cancel`}</button>
-          <button
-            className={cx("Button ml2", {
-              "Button--primary": isValid,
-            })}
-            onClick={handleCommit}
-            disabled={!isValid}
-          >
+          <Button primary={isValid} disabled={!isValid} onClick={handleCommit}>
             {initialName ? t`Update` : t`Done`}
-          </button>
-        </div>
-        <div>
+          </Button>
+
           {initialName && onRemoveExpression ? (
-            <a
-              className="pr2 ml2 text-error link"
+            <RemoveLink
+              as="a"
+              href="#"
```

Comment:
First of all, I don't think we've ever used `href="#"` before.
Secondly, I couldn't find a way to render this `RemoveLink`. Seems `onRemoveExpression` is only passed when we're rendering [`ExtendedOptions`](https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/frontend/src/metabase/query_builder/components/ExtendedOptions.jsx#L167).  Which only render if [`features.sort` or `features.limit` is true](https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/frontend/src/metabase/query_builder/components/ExtendedOptions.jsx#L215-L217). 

And they'll only be true when it's rendered in [`AuditQueryDetail` only](https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQueryDetail.jsx#L103) since it doesn't pass any `features` prop causing it to use the [defaultProps](https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/frontend/src/metabase/query_builder/components/GuiQueryEditor.jsx#L38-L47) instead.

The other 2 places
- https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/frontend/src/metabase/admin/datamodel/components/MetricForm/MetricForm.tsx#L65
- https://github.com/metabase/metabase/blob/2d9c6eb9e839b446ce988d4b6ded372d6dc78bbf/frontend/src/metabase/admin/datamodel/components/SegmentForm/SegmentForm.tsx#L64
don't pass `features` that has either `sort: true` or `limit: true`, so it won't render this `RemoveLink` for sure.

From that point, we can trace that we will render `AuditQueryDetail` when the path is `/admin/audit/query/:queryHash`.

But the only way to go to that page is by clicking the database in `/admin/audit/databases/all` and click on any ad-hoc query. Here, the page will crash, but we can [remove this line to circumvent this bug](https://github.com/metabase/metabase/blob/e92dc65fa94429dd5f737943d98e7a47e17ba3fa/enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQueryDetail.jsx#L117) (I'm not sure why it crashes, but that a different story).

From there we'll finally see the `GuiQueryEditor` but it has `readOnly` props and isn't seemed to be clickable at all.
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/1937582/222540604-1a9bbb72-7877-4f98-9efb-0cc64cf82fe7.png">


In conclusion, I don't think this code will ever get called. Maybe in the future, we could remove this `<RemoveLink />` from this component.

But back to the topic at hand, since I don't think this code get called, you could either leave code as is (before the change) or remove this `href="#"`.